sources/api/ecs-settings-applier/src/ecs.rs

```diff
@@ -112,7 +113,7 @@ async fn run() -> Result<()> {
         image_pull_behavior: ecs
             .image_pull_behavior
             .as_ref()
-            .map(|b| ECSImagePullBehavior::try_from(b.as_ref()).unwrap() as u8),
+            .map(|b| b.parse::<ECSImagePullBehavior>().unwrap() as u8),
```

Comment:
This function looks difficult to unit test. Do you know if it's covered by an integration test of some sort?

Reply:
I think the test I will execute is to actually run Bottlerocket and use a feature that utilizes `ECSImagePullBehavior`.

Reply:
Upon further investigation, `ECSImagePullBehavior` is not directly used -- `ECSAgentImagePullBehavior` is what is used. I say that because that is what is used in `ECSSettings`: 

https://github.com/bottlerocket-os/bottlerocket/blob/fa341b494872a4228a43de9c0ee95bc80f8553ba/sources/models/src/lib.rs#L238

That `ECSAgentImagePullBehavior` code is tested here:

https://github.com/bottlerocket-os/bottlerocket/blob/develop/sources/models/src/modeled_types/ecs.rs#L275-L293